cmd/juju/commands/main.go

```diff
@@ -218,17 +218,40 @@ func juju2xConfigDataExists() bool {
 
 // NewJujuCommand ...
 func NewJujuCommand(ctx *cmd.Context) cmd.Command {
-	jcmd := jujucmd.NewSuperCommand(cmd.SuperCommandParams{
-		Name:                "juju",
-		Doc:                 jujuDoc,
-		MissingCallback:     RunPlugin,
+	var jcmd *cmd.SuperCommand
+	jcmd = jujucmd.NewSuperCommand(cmd.SuperCommandParams{
+		Name: "juju",
+		Doc:  jujuDoc,
+		MissingCallback: RunPlugin(func(ctx *cmd.Context, subcommand string, args []string) error {
+			if cmdName, _, ok := jcmd.FindClosestSubCommand(subcommand); ok {
```

Comment:
Nice. Does this use levenshtein distance to find the closest command?

Reply:
Indeed it does, but we cap the result so that matches with large deviation don't result in large mismatches.